### ISSUE TITLE
How to find out the https proxy？

### ISSUE THREAD

**Issue by hzbd** [ricerati/proxy-checker-python#6]
It seems that there is no distinction between http and https at the moment.

**Comment by ricerati** [comment]
You're correct. I'll add a test for HTTPS proxies during the following days.

**Comment by pocupine** [comment]
We are looking forward to this new function. :)

**Comment by ma-ji** [comment]
Has this been done yet? Looking forward to it!

**Comment by Scolymus** [comment]
Hi, I made a new version which checks this, but I'm not sure if it's correctly implemented. If someone has one https proxy that works correctly (nothing free from internet) it would be very nice to check if it works correctly.

Check https://github.com/Scolymus/proxy-checker-python
